# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1331933907 25200
#      Fri Mar 16 14:38:27 2012 -0700
# Node ID 00d8118d240153db1e19ade2da998840d141f770
# Parent  f49f14f006488ede56c5e652c1d9d0dd1dac97e3
maint: Remove issuperuser.m from private function list for pkg directory.

* pkg/module.mk: Remove issuperuser.m from private function list for pkg directory.

diff --git a/scripts/pkg/module.mk b/scripts/pkg/module.mk
--- a/scripts/pkg/module.mk
+++ b/scripts/pkg/module.mk
@@ -18,17 +18,16 @@ pkg_PRIVATE_FCN_FILES = \
   pkg/private/get_forge_pkg.m \
   pkg/private/getarch.m \
   pkg/private/getarchdir.m \
   pkg/private/getarchprefix.m \
   pkg/private/get_unsatisfied_deps.m \
   pkg/private/install.m \
   pkg/private/installed_packages.m \
   pkg/private/is_architecture_dependent.m \
-  pkg/private/issuperuser.m \
   pkg/private/list_forge_packages.m \
   pkg/private/load_package_dirs.m \
   pkg/private/load_packages.m \
   pkg/private/load_packages_and_dependencies.m \
   pkg/private/packinfo_copy_file.m \
   pkg/private/parse_pkg_idx.m \
   pkg/private/prepare_installation.m \
   pkg/private/print_package_description.m \
